### ISSUE TITLE
Travis build stalls while compiling

### ISSUE THREAD

**Issue by jedesah** [quasar-analytics/quasar#1405]
It appears that sometimes the Travis build stalls during the compiling phase with these being the last few lines being printed out by the build:

```
[info] Done updating.
[info] Compiling 18 Scala sources to /home/travis/build/quasar-analytics/quasar/foundation/target/scala-2.11/classes...
[info] Compiling 9 Scala sources to /home/travis/build/quasar-analytics/quasar/effect/target/scala-2.11/classes...
[info] Compiling 5 Scala sources to /home/travis/build/quasar-analytics/quasar/foundation/target/scala-2.11/test-classes...
[info] Compiling 5 Scala sources to /home/travis/build/quasar-analytics/quasar/js/target/scala-2.11/classes...
[info] Compiling 3 Scala sources to /home/travis/build/quasar-analytics/quasar/ejson/target/scala-2.11/classes...
[info] Compiling 2 Scala sources to /home/travis/build/quasar-analytics/quasar/ejson/target/scala-2.11/test-classes...
[info] Compiling 91 Scala sources to /home/travis/build/quasar-analytics/quasar/core/target/scala-2.11/classes...
[info] Compiling 1 Scala source to /home/travis/build/quasar-analytics/quasar/js/target/scala-2.11/test-classes...

No output has been received in the last 10 minutes, this potentially indicates a stalled build or something wrong with the build itself.
The build has been terminated
```

Since not all sub-projects finished compiling, it seems either Travis is just being way too slow or the Scala compiler is hanging.

If the former, we may want to consider using [travis_wait](https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received) in order to afford the build more time to compile.

**Comment by mossprescott** [comment]
For what it's worth, I've seen this happen a lot where it turned out that the compile was finishing, and SBT was running tests, and the *test* was stalled. Since there's no output until at least one entire spec finishes, it looks like the compile is what's stuck. Of course I can't say if that's what's happening here.

**Comment by alissapajer** [comment]
@mossprescott @jedesah should we mark this as closed, now that #1459 has landed?

**Comment by mossprescott** [comment]
I'd say so, unless anyone's seen it happen since merging.

**Comment by alissapajer** [comment]
Right, we can wait until the batch of like 10 open PRs all make it in, up to say the most recent #1460, and then if those make it in without seeing a hang, we'll call it fixed.

**Comment by jedesah** [comment]
I'll just go ahead and close it now. Can always re-open it if it occurs again.

**Comment by alissapajer** [comment]
@sellout and I are seeing this again. We are experimenting with `travis_wait`.

**Comment by alissapajer** [comment]
This is the PR we saw hangs in: https://github.com/quasar-analytics/quasar/pull/1427

**Comment by alissapajer** [comment]
Closing again - hopefully fixed when #1427 merged with `travis_wait` included.
